src/test/java/uk/gov/justice/digital/delius/service/SpgNotificationServiceTest.java

```diff
@@ -585,6 +586,107 @@ public void offenderIdIsSavedWithNotification() {
             assertThat(notification.getParentEntityId()).isNull();
         }
 
+    }
+    @Nested
+    class InsertAdditionalOffenderIdentifier {
+        @Test
+        public void withOneInterestedCRCNotificationIsSaved() {
+            when(spgNotificationHelperRepository.getInterestedCRCs(any())).thenReturn(ImmutableList.of(ProbationArea
+                    .builder()
+                    .code("AA")
+                    .build()));
+
+            spgNotificationService.notifyInsertOfOffenderAdditionalIdentifier(
+                    Offender
+                            .builder()
+                            .offenderId(99L)
+                            .build(),
+                    AdditionalIdentifier
+                            .builder()
+                            .additionalIdentifierId(88L)
+                            .build());
+
+            verify(spgNotificationRepository, atLeastOnce()).saveAll(spgNotificationsCaptor.capture());
```

Comment:
I was just wondering why these tests all have `atLeastOnce()`... is it just that the test doesn't care about which areas have an interest?